pkg/api/handlers/libpod/containers.go

```diff
@@ -117,19 +131,13 @@ func KillContainer(w http.ResponseWriter, r *http.Request) {
 }
 
 func WaitContainer(w http.ResponseWriter, r *http.Request) {
-	_, err := utils.WaitContainer(w, r)
+	exitCode, err := utils.WaitContainer(w, r)
 	if err != nil {
 		utils.InternalServerError(w, err)
 		return
 	}
-	utils.WriteResponse(w, http.StatusNoContent, "")
-}
-
-func PruneContainers(w http.ResponseWriter, r *http.Request) {
-	// TODO Needs rebase to get  filers; Also would be handy to define
-	// an actual libpod container prune method.
-	// force
-	// filters
+	// TODO can we straight up return an int32 like this ?
+	utils.WriteResponse(w, http.StatusOK, exitCode)
```

Comment:
Do you really want the client to really get an integer or a string with the value of the integer?

Reply:
integer i believe.

Reply:
Then how are you dealing with endianness?

Reply:
switched to string